src/main/resources/biz_en.properties

```diff
@@ -756,6 +748,15 @@ Packages.permission.name = Name
 Packages.upgrades = Upgrades
 Parameter.invalidValue = Error in parameter ${name}: ${message}
 Parameter.required = The parameter ${name} must be filled.
+PasswordSettings.fine.description = Passwords consisting of uppercase/lowercase letters or digits.
+PasswordSettings.fine.label = Moderately Secure
+PasswordSettings.insufficient.description = Passwords shorter than ${nextMinLength} characters.
+PasswordSettings.insufficient.label = Insufficient
+PasswordSettings.secure.description = Passwords consisting of letters and special characters.
+PasswordSettings.secure.label = Secure
+PasswordSettings.secure.labelAddition = recommended
```

Comment:
Lowercase?

Reply:
Ist beim deutschen Key hier auch so...

Reply:
Von wem kommt der neue Key? @fhaScireum?

Reply:
Yes @fhaScireum

Reply:
Der Text  steht klein in einer Klammer, wie bisher auch. Dass passt.